postgres/schema.sql

```diff
@@ -1,4 +1,5 @@
 CREATE TABLE IF NOT EXISTS responses (tx_id uuid PRIMARY KEY, ts timestamp WITH time zone DEFAULT NOW(), invalid boolean DEFAULT NULL, data jsonb);
+CREATE TABLE IF NOT EXISTS feedback_responses (id SERIAL, ts timestamp WITH time zone DEFAULT NOW(), invalid boolean DEFAULT NULL, data jsonb, survey char(25), period char(25));
```

Comment:
Use an `@property` and validate the string length in an `@property.setter` method, raising an exception if necessary